### ISSUE TITLE
feat: Remove add friend limitation

### ISSUE THREAD

**Issue by CanerKaraca23** [rhunk/SnapEnhance#134]
### Description

Remove daily add friend limitation and add a button to add all friend suggestions.

It will be useful for flirting lol.

### Agreement

- [X] There is no issue already suggesting my feature.

**Comment by TheVisual** [comment]
No possible its server sided lol

**Comment by CanerKaraca23** [comment]
> Not possible its server sided lol

Are you sure? Really, if Snapchat+ isn't server sided and this limitation is server sided, then Snapchat is stupid.

**Comment by TheVisual** [comment]
@CanerKaraca23 yes it is indeed server sided and binded to the account specifically.
